appscale/tools/appscale.py

```diff
@@ -85,7 +92,7 @@ class AppScale():
   get <regex>                       Gets all AppController properties matching
                                     the provided regex: for developers only.
   help                              Displays this message.
-  init <cloud|cluster>              Writes a new configuration file for
+  init [cloud|cluster]              Writes a new configuration file for
```

Comment:
This branch does not remove any of the previous templates or functionality. 
I believe it should be a separate pull after 3.2.